derp/derp_server.go

```diff
@@ -1955,6 +1955,22 @@ func (s *Server) ConsistencyCheck() error {
 			s.curClients.Value(),
 			len(s.clients)))
 	}
+	if s.verifyClientsLocalTailscaled {
+		ctx, cancel := context.WithTimeout(context.Background(), time.Second)
```

Comment:
is second sufficient for the two calls? could `Status` be soo large that it fails?

Reply:
on a local socket? perhaps, seems unlikely though?

Reply:
I'm sure with large number of peers we would trigger this in just the JSON encoding/decoding

Reply:
Upped to 5s.